mps_youtube/listview.py

```diff
@@ -0,0 +1,237 @@
+"""
+    DOCSTING COMES HERE
+"""
+import re
+
+from . import c, util, content
+
+
+class ListViewItem(object):
```

Comment:
Mpsyt is now Python 3 only, so explicitly inheriting from `object` is redundant.